code/index.js

```diff
@@ -98,7 +98,16 @@ exports.handler = async function (event) {
     signer
   );
 
-  const state = await GovernorContract.state(proposalId);
+  //Check proposal state, if proposalId doesn't exist yet, just skip
+  let state;
+  try {
+    state = await GovernorContract.state(proposalId);
```

Comment:
Just double-check it works with decimal `proposalId` (I think it does), but since we removed the `hexlify` in the creation script, here we'll have the `proposalId` hardcoded in decimal.

In case it's not working, just `hexlify` it here in the autotask code